### DISCUSSION TITLE
Validators

### DISCUSSION THREAD

**Discussion by ddibert** [davetapley/helium-tax#40] [Ideas]
Could validators be added?

**Comment by davetapley** [comment]
Oh yes, I suppose so!
I'll take a look when I get a chance.

Also if anyone is reading this and wants to make a PR, you're very welcome 😁 

I _assume_ it would be as simple as getting validators here:
https://github.com/davetapley/helium-tax/blob/de72d773917c3176b9238196438262885a0c5f8e/tax.js#L16-L17

Then modifying `getRow` to handle validators **and** hotspots here:
https://github.com/davetapley/helium-tax/blob/de72d773917c3176b9238196438262885a0c5f8e/tax.js#L78-L80

**Comment by lstew48** [comment]
Hey Dave, just dropping by to see if validator info will be coming soon. Need it for them taxes! TY for your work!

**Reply by davetapley** [reply]
Hi, sorry, I'm unlikely to get to it any time soon.

I know there are other solutions out there which might support it, maybe https://www.minertax.com/?
